src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -284,6 +290,7 @@ public void execute(BuildResult buildResult) {
             }
 
             if (isGitLab(providers)) {
+                buildScan.value("CI provider", "GitLab CI");
```

Comment:
I would take away the `CI` part.

Reply:
Just to be clear, make it only `GitLab`, but keep the `CI provider` value key, correct?

Reply:
@ribafish Yes.